Makefile

```diff
@@ -1,3 +1,3 @@
 test:
-	@flake8 . --exclude fabfile.py --ignore=E241,E501,F403 # FIXME: Fix these errors and stop ignoring them
+	@flake8 . --exclude fabfile.py,venv --ignore=E241,E501,F403 # FIXME: Fix these errors and stop ignoring them
```

Comment:
@alexmuller I agree, I wasn't sure about that. I've removed that commit from the PR.